<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>configDlg</class>
 <widget class="QDialog" name="configDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Config serial port</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>311</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Serial Port</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>131</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Speed</string>
   </property>
   <widget class="QRadioButton" name="rb9600">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>24</y>
      <width>114</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>9600</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb19200">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>44</y>
      <width>114</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>19200</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb38400">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>64</y>
      <width>114</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>38400</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb57600">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>84</y>
      <width>114</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>57600</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb115200">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>104</y>
      <width>114</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>115200</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb230400">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>114</width>
      <height>10</height>
     </rect>
    </property>
    <property name="text">
     <string>230400</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb460800">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>114</width>
      <height>10</height>
     </rect>
    </property>
    <property name="text">
     <string>460800</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb921600">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>114</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>921600</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="cbOpenStart">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>181</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Open on Startup</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbHwFlow">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>291</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Use hardware flow control</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listPorts">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>151</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="labelStop">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboStop">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>69</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>69</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>199</x>
     <y>210</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="labelParity">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Parity</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboParity">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>69</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>69</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="labelData">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Data</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboData">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>69</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>69</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>configDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>configDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
